macOS/DuckDuckGoVPN/VPNProxyLauncher.swift

```diff
@@ -47,24 +47,35 @@ final class VPNProxyLauncher {
 
     // MARK: - Status Changes
 
+    private struct ProxyStatusUpdate: Equatable {
+        let newStatus: NEVPNStatus
+        let isProxyStatusChange: Bool
+    }
```

Comment:
Created this structure to be able to remove duplicate VPN and proxy status updates.

This means we won't get the same status update twice which breaks the loop I was seeing.